fagiServer/src/Server.java

```diff
@@ -27,7 +27,7 @@ public Server(int port) {
         this.port = port;
         try {
             String name = "test";
-            String ip = getExternalIP();
+            String ip = "127.0.0.1"; //getExternalIP();
```

Comment:
I don't know where this came from. But it needs to go.

Reply:
I don't like it to be in the config file as we are kindof starting the server in debug mode and the config file is generated after this code so it would not work either way. Maybe a --debug argument? Should we make this as a bug on scrumdesk?

Reply:
Yeah I think we should do that. It sounds like a quick feature. But let's leave it for after our tests.

Reply:
If you could add it, resolve this conversation. And continue reviewing/approving the PR so we can get it in.

Reply:
I will :)